apps/export/entries/report_exporter.py

```diff
@@ -55,19 +55,167 @@ class ExportDataVersionMismatch(Exception):
     pass
 
 
+class WidgetExporter:
+
+    @staticmethod
+    def _add_common(para, text, bold):
+        para.add_run(text, bold)
+
+    @staticmethod
+    def _add_scale_widget_data(para, label, color, bold):
```

Comment:
Make sure return is missing or its intentionally done.

Reply:
Return is not required here. This function is using para to add attributes to the document.